### PR TITLE
Remove specific code for old devices and UWP

### PR DESCRIPTION
**Pull request by alexander-kotov-dx**

### CODE REVIEW

**Review by San4es — DISMISSED** [review]
LGTM

**Review by mrjohn0011 — DISMISSED** [review]

**Review by RomanTsukanov — DISMISSED** [review]

File: js/core/devices.js [hunk 1]
```diff
@@ -24,10 +24,8 @@ var KNOWN_UA_TABLE = {
     "androidPhone": "Android Mobile",
     "androidTablet": "Android",
     "win8": "MSAppHost",
```
Comment (BingoRUS) [inline js/core/devices.js]: How about `win8` and `win10`? They look redundant here because you removed `win` from uaParsers.

File: js/ui/toast.js [hunk 2]
```diff
@@ -217,6 +183,10 @@ var Toast = Overlay.inherit({
                     */
                     closeOnOutsideClick: true,
 
+                    /**
+                    * @name dxToastOptions.width
+                    * @default 'auto' @for Android
+                    */
```
Comment (BingoRUS) [inline js/ui/toast.js:189]: Is there any reason for the new doc comment here?

File: testing/tests/DevExpress.ui.widgets/popup.tests.js [hunk 3]
```diff
@@ -196,27 +196,11 @@ QUnit.test("done button is located after cancel button in non-win8 device", func
...
-    assert.equal($popupBottom.text(), "OKCancel", "buttons order is correct");
-
-    instance.option("toolbarItems", [{ shortcut: "done" }, { shortcut: "cancel" }]);
-    $popupBottom = instance.$content().parent().find(".dx-popup-bottom");
-    assert.equal($popupBottom.text(), "OKCancel", "buttons order is correct");
+    devices.current(devices.real());
 });
 
 QUnit.test("buttons should be rendered correctly after toolbar was repainted", function(assert) {
+    devices.current("desktop");
```
Comment (BingoRUS) [inline testing/tests/DevExpress.ui.widgets/popup.tests.js:203]: Why didn't we need this before?

File: testing/tests/DevExpress.ui.widgets/scrollView.tests.js [hunk 4]
```diff
@@ -2133,166 +2125,6 @@ QUnit.test("scroll fires with correctly arguments", function(assert) {
...
-    var $reachBottom = $scrollView.find("." + SCROLLVIEW_BOTTOM_POCKET_CLASS);
-
-    assert.ok($reachBottom.is(":hidden"), "reach bottom is hidden");
-
-    $scrollView.dxScrollView("option", "onReachBottom", noop);
-
-    assert.ok($reachBottom.is(":visible"), "reach bottom is hidden");
-});
-
-QUnit.test("release fires update", function(assert) {
```
Comment (BingoRUS) [inline testing/tests/DevExpress.ui.widgets/scrollView.tests.js:2223]: This test doesn't look related to the old devices, why are we going to remove it?

File: js/ui/toast.js (hunk 2 above)
Comment (alexander-kotov-dx) [reply js/ui/toast.js:189]: I am removing doc comment in default options for windows device. But the part for Android is still actual.
```
                    /**
                    * @name dxToastOptions.width
                    * @default 'auto' @for Android|Windows_10_Mobile
                    */
```

Comment (BingoRUS) [reply js/ui/toast.js:189]: Oh, got it now. Thanks for the clarification.

File: testing/tests/DevExpress.ui.widgets/popup.tests.js (hunk 3 above)
Comment (alexander-kotov-dx) [reply testing/tests/DevExpress.ui.widgets/popup.tests.js:203]: In the old version, we use current devices "win" set in the previous test. I have fixed tests to use actual devices, but these tests failed for some devices because some devices have another buttons order by default.

File: testing/tests/DevExpress.ui.widgets/scrollView.tests.js (hunk 4 above)
Comment (alexander-kotov-dx) [reply testing/tests/DevExpress.ui.widgets/scrollView.tests.js:2223]: This test is used for the removed slideDown strategy. There are some analogs of this test for other refresh strategies.

**Review by AlexJameson — APPROVED** [review]

**Review by BingoRUS — APPROVED** [review]
